CONTRIBUTORS.md

```diff
@@ -59,3 +59,4 @@ Non-JPL Contributors to the F' Software Framework:
 * keck-in-space
 * sommercharles
 * SterlingPeet
+* arijitdas123student
```

Comment:
oh, and follow the bot's instructions ;-)

Reply:
@jsoref Do I only need to paste the instructions laid by the bot as a comment for the check the complete?

Reply:
that'll work, yes, or you can do the equivalent work by hand if you don't trust the bot.

Reply:
actually, you might as well do it by hand, otherwise your PR will have a bunch of unrelated cleanup...

just add `arijitdas` to the `expect.txt` file alphabetically...

Reply:
(And the other things the bot doesn't like that you've added...)

Reply:
oh nice....done!